ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/TestBlock.hs

```diff
@@ -543,7 +545,19 @@ testInitExtLedgerWithState st = ExtLedgerState {
     , headerState = genesisHeaderState ()
     }
 
-type instance LedgerCfg (LedgerState (TestBlockWith ptype)) = HardFork.EraParams
+data TestBlockLedgerConfig = TestBlockLedgerConfig {
+  hardForkParams :: HardFork.EraParams,
+  -- | `Nothing` means an infinite forecast range.
+  -- Instead of SlotNo, it should be something like "SlotRange"
+  forecastRange  :: Maybe SlotNo
```

Comment:
We usually try to avoid giving record fields "generic" names as they could otherwise "pollute" the namespace (GHC 8.10 doesn't support eg `NoFieldSelectors` yet).
```suggestion
  testBlockEraParams :: HardFork.EraParams,
  -- | `Nothing` means an infinite forecast range.
  -- Instead of SlotNo, it should be something like "SlotRange"
  testBlockForecastRange  :: Maybe SlotNo
```

Reply:
Is it better to prefix with `test` or `tblc`?

Reply:
Done with `tblc` in 90df5b3ad291695e0d4684f136a834df8f9e0008.